### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by olya-yevsieienko**

### CODE REVIEW

**Review by YarinKiri2l — APPROVED** [review]
Good Job!

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,17 @@
...
-  // write code here
+  const rentalDays = 40 * days;
+
+  if (days >= 7) {
+    return rentalDays - 50;
+  }
+
+  if (days >= 3) {
+    return rentalDays - 20;
+  }
```
Comment (illiarerun) [inline src/calculateRentalCost.js:39]: [Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md?rgh-link-date=2022-06-06T18%3A50%3A47Z) 1st item. You should create variables for main values - 20, 50, 40, 3, 7 they shouldn't be just magic numbers

**Review by illiarerun — CHANGES_REQUESTED** [review]
GJ, let's work on fixes now:

**Comment by olya-yevsieienko** [comment]
I got it, thanks!

пн, 6 июн. 2022 г. в 22:39, Illia Vovchenko ***@***.***>:

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalDay = 40;
+  const rentalDays = rentalDay * days;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
```
Comment (illiarerun) [inline src/calculateRentalCost.js:34]: [Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 5th item, you shouldn't hardcode any values in variable names

**Review by illiarerun — CHANGES_REQUESTED** [review]

**Review by illiarerun — APPROVED** [review]
GJ 🔥

**Pull request closed without merging**